addons/website_event_track/migrations/11.0.1.0/post-migration.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2019 Eficent <http://www.eficent.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from psycopg2.extensions import AsIs
+from openupgradelib import openupgrade
+
+
+def fill_event_track_stage_id(env):
+    openupgrade.logged_query(
+        env.cr, """
+        UPDATE event_track et
+        SET stage_id = ets.id
+        FROM event_track_stage ets
+        WHERE et.%s = lower(ets.name)
```

Comment:
pending to check how to do with people that have the translated names for the stages, because `name` field has translations=True.

Reply:
Fixed using XML-IDs